acceptance-tests/src/test/resources/features/site/indicator.feature

```diff
@@ -0,0 +1,44 @@
+
+Feature: National Indicator summary page
+
+    As a site user I should see a summary page of all the Indicator information
+    so I can view each and assess the usefulness of the indicator.
+
+    Scenario: Indicator Summary page - fields
+        Given I navigate to the "sample-indicator" page
+        Then I should see a page titled "People with stroke admitted to an acute stroke unit within 4 hours of arrival at hospital"
+        Then I can see element with id "assured-status" with content "Independently assured by IGB"
```

Comment:
Is there a reason you've decided not to use the uipath instead of id to keep it consistent with the publications stuff?

Reply:
I was seeking a solution that didn't involve putting lots of 'ps.document....' all over the nil stuff

Reply:
You can use the same works as in the id if you want to but just stick it in `data-uipath` instead of `id`